extensions/ocdav/pkg/command/ocdav.go

```diff
@@ -144,28 +144,3 @@ func (s OCDavSutureService) Serve(ctx context.Context) error {
 
 	return nil
 }
-
-// loadUserAgent reads the user-agent-whitelist-lock-in, since it is a string flag, and attempts to construct a map of
-// "user-agent":"challenge" locks in for Reva.
-// Modifies cfg. Spaces don't need to be trimmed as urfavecli takes care of it. User agents with spaces are valid. i.e:
-// Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:83.0) Gecko/20100101 Firefox/83.0
-// This function works by relying in our format of specifying [user-agent:challenge] and the fact that the user agent
-// might contain ":" (colon), so the original string is reversed, split in two parts, by the time it is split we
-// have the indexes reversed and the tuple is in the format of [challenge:user-agent], then the same process is applied
-// in reverse for each individual part
-func loadUserAgent(c *cli.Context, cfg *config.Config) error {
```

Comment:
since user-agent-whitelist-lock-in is never set in the context the code is dead. if the `Middleware.Auth.CredentialsByUserAgent` need to be set this is best done via a yaml file